models/_sector/dex/trades/scroll/_schema.yml

```diff
@@ -72,3 +72,20 @@ models:
             - evt_index
       - check_dex_base_trades_seed:
           seed_file: ref('scrollswap_scroll_base_trades_seed')
+
+  - name: syncswap_v1_scroll_base_trades
```

Comment:
We may need to add `syncswap_v1_scroll_base_trades` into `dex_scroll_base_trades`:
https://github.com/duneanalytics/spellbook/blob/main/models/_sector/dex/trades/scroll/dex_scroll_base_trades.sql

Reply:
that makes sense, ive added it now